src/mame/drivers/galaxian.cpp

```diff
@@ -6099,14 +6099,13 @@ MACHINE_CONFIG_START(galaxian_state::konami_sound_2x_ay8910)
 MACHINE_CONFIG_END
 
 
-void galaxian_state::scramble_base(machine_config &config)
-{
+MACHINE_CONFIG_START(galaxian_state::scramble_base)
 	konami_base(config);
 	konami_sound_2x_ay8910(config);
 
 	/* blinking frequency is determined by 555 counter with Ra=100k, Rb=10k, C=10uF */
-	TIMER(config, "stars").configure_periodic(FUNC(galaxian_state::scramble_stars_blink_timer), PERIOD_OF_555_ASTABLE(100000, 10000, 0.00001));
-}
+	MCFG_TIMER_DRIVER_ADD_PERIODIC("stars", galaxian_state, scramble_stars_blink_timer, PERIOD_OF_555_ASTABLE(100000, 10000, 0.00001))
+MACHINE_CONFIG_END
```

Comment:
Thanks, but these reverts some recent changes and will cause compile to fail.

Reply:
Updated, thanks and sorry.